tests/ordered_unique.rs

```diff
@@ -63,15 +63,15 @@ fn test_modify_by_field1() {
     };
     map.insert(elem1);
     map.insert(elem2);
-
+    {
     let mut field_1_iter = map.iter_by_field1();
```

Comment:
format this file.

Reply:
[2bff67e](https://github.com/lun3x/multi_index_map/pull/23/commits/2bff67eec2587697fc6088cd8e6a6bd722e992e2)